android/langid/app/src/main/java/com/google/mlkit/samples/languageid/EntryChoiceActivity.kt

```diff
@@ -19,7 +19,7 @@ package com.google.mlkit.samples.languageid
 import android.content.Intent
 import com.mlkit.example.internal.BaseEntryChoiceActivity
 import com.mlkit.example.internal.Choice
-import com.google.mlkit.samples.languageid.java.MainActivity
+import com.google.mlkit.samples.languageid.java.LangIdMainActivity
```

Comment:
I feel only xml file renaming is required because package name can help distinguish activity classes with same name.

Reply:
I reverted to only xml renames, and keep Java/Kotlin classes